### PR TITLE
feature(ui-ux): ocean Interface v2.0

### PR DESCRIPTION
**Pull request by G9000**
#### What kind of PR is this?:
/kind feature

#### What this PR does / why we need it:
Update Ocean Interface to version LW 2.0

#### Which issue(s) does this PR fixes?:
Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*

### CODE REVIEW

File: mobile-app/app/components/OceanInterface/TransactionDetail.tsx [hunk 1]
```diff
@@ -2,48 +2,65 @@ import { View } from 'react-native'
...
   onClose,
-  title
+  title,
+  transactionStatusCode
 }: TransactionDetailProps): JSX.Element {
   title = title ?? translate('screens/OceanInterface', 'Broadcasting...')
+
+  const { isLight } = useThemeContext()
+
+  const themedOrange = isLight ? 'border-warning-500' : 'border-darkwarning-500'
```
Comment (pierregee) [inline mobile-app/app/components/OceanInterface/TransactionDetail.tsx:31]: can make use of tailwind directly + themed component to lessen the ternary operator logic
``` light={tailwind('bg-mono-light-v2-00', {
        'border-warning-500': <condition_here>
        'border-success-500': <condition_here>
      })}

File: mobile-app/app/components/OceanInterface/TransactionDetail.tsx [hunk 2]
```diff
@@ -2,48 +2,65 @@ import { View } from 'react-native'
...
+      light={tailwind('bg-mono-light-v2-00')}
+      style={tailwind(`w-full rounded-lg-v2 px-5 flex flex-row py-3 items-center border-0.5 ${borderColour}`)}
+    >
       {
         !broadcasted
           ? <ThemedActivityIndicator />
           : (
             <ThemedIcon
-              dark={tailwind('text-darksuccess-500')}
+              dark={tailwind(transactionStatusCode === 200 ? 'text-darksuccess-500' : 'text-darkwarning-500')}
```
Comment (pierregee) [inline mobile-app/app/components/OceanInterface/TransactionDetail.tsx:47]:
```suggestion
              dark={tailwind({
             'text-darksuccess-500': transactionStatusCode === 200,
             'text-darkwarning-500':  transactionStatusCode !== 200
              }
              )}
```

File: mobile-app/app/components/OceanInterface/OceanInterface.tsx [hunk 3]
```diff
@@ -118,17 +116,21 @@ export function OceanInterface (): JSX.Element | null {
...
           }
           let title
+          let transactionStatusCode
           try {
             await waitForTxConfirmation(transaction.tx.txId, client, logger)
             title = 'Transaction completed'
+            transactionStatusCode = 200
           } catch (e) {
             logger.error(e)
             title = 'Sent but not confirmed'
```
Comment (pierregee) [inline mobile-app/app/components/OceanInterface/OceanInterface.tsx:126]: image

**Review by pierregee — CHANGES_REQUESTED** [review]

File: mobile-app/app/components/OceanInterface/TransactionDetail.tsx (hunk 1 above)
Comment (G9000) [reply mobile-app/app/components/OceanInterface/TransactionDetail.tsx]: @pierregee done try check hahaa

File: mobile-app/app/components/OceanInterface/OceanInterface.tsx [hunk 4]
```diff
@@ -118,17 +116,21 @@ export function OceanInterface (): JSX.Element | null {
             transaction.onBroadcast()
           }
           let title
+          let transactionStatusCode
           try {
             await waitForTxConfirmation(transaction.tx.txId, client, logger)
-            title = 'Transaction completed'
+            title = 'Transaction confirmed'
+            transactionStatusCode = 200
```
Comment (thedoublejay) [inline mobile-app/app/components/OceanInterface/OceanInterface.tsx:123]: Let's make this an enum

File: mobile-app/app/components/OceanInterface/TransactionDetail.tsx [hunk 5]
```diff
@@ -12,38 +12,61 @@ interface TransactionDetailProps {
   txUrl?: string
   onClose: () => void
   title?: string
+  transactionStatusCode?: number
```
Comment (thedoublejay) [inline mobile-app/app/components/OceanInterface/TransactionDetail.tsx:15]: Enum of status codes

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Review by thedoublejay — CHANGES_REQUESTED** [review]
1. View on scan button too close to the tx id. Add margin left
2. Increase size of close button

File: mobile-app/app/components/OceanInterface/OceanInterface.tsx (hunk 4 above)
Comment (G9000) [reply mobile-app/app/components/OceanInterface/OceanInterface.tsx]: @thedoublejay done check if its up to standard haha

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**